src/hooks/cakeVault/useGetVaultBountyInfo.ts

```diff
@@ -1,33 +1,23 @@
-import { useState, useEffect } from 'react'
-import BigNumber from 'bignumber.js'
-import { useGetApiPrice } from 'state/hooks'
-import { useCakeVaultContract } from 'hooks/useContract'
+import { useState, useEffect, useMemo } from 'react'
+import { useCakeVault, useGetApiPrice } from 'state/hooks'
 import { getFullDisplayBalance } from 'utils/formatBalance'
-import makeBatchRequest from 'utils/makeBatchRequest'
 import { getCakeAddress } from 'utils/addressHelpers'
+import BigNumber from 'bignumber.js'
+
+const useGetVaultBountyInfo = () => {
+  const {
+    estimatedCallBountyReward: estimatedCallBountyRewardAsString,
+    totalPendingCakeRewards: totalPendingCakeRewardsAsString,
+  } = useCakeVault()
```

Comment:
Get these values from redux. 

Some calcs to get the $ bounty & CAKE value are still being done in this hook. This will change once this PR is merged, it would be great to get reviews on it https://github.com/pancakeswap/pancake-frontend/pull/1054 🙏